server/views/pages/additional-learning-needs-screener/add-challenges/index.njk

```diff
@@ -7,26 +7,36 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full-width govuk-!-padding-left-3 govuk-!-padding-right-3">
 
-      <h1 class="govuk-heading-l">Select the challenges identified by the screener</h1>
-
       <form class="form additional-learning-needs-form" method="post" novalidate="">
         <div class="govuk-form-group">
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
           <div class="govuk-form-group">
-            <div class="govuk-checkboxes" data-module="govuk-checkboxes">
+            {% set fieldError = errors | findError('challengeTypeCodes')  %}
```

Comment:
Because we are marking up the checkboxes ourselves (rather than using the govuk macro) we also need to handle and markup the error messages/styling ourselves